ott/geometry/costs.py

```diff
@@ -25,6 +26,37 @@
 from ott.geometry import matrix_square_root
 
 
+def x_to_means_and_covs(x, dimension):
```

Comment:
For readability, I would move these 3 functions to the bottom of the file, so that the file starts with the main element, the definition of `CostFn`

Reply:
sure ok. that will look nicer since those are really utility functions.

I could also move them to a different file (as utility functions) like I have proposed before